formats/filtered/src/Filtered.php

```diff
@@ -164,6 +165,7 @@ protected function getResultText( SMWQueryResult $res, $outputmode ) {
 		$result = [];
 		while ( $row = $res->getNext() ) {
 			$result[ $this->uniqid() ] = new ResultItem( $row, $this );
+			usleep( 1 );
```

Comment:
Don't like it either. The problem is, no matter what I try I cannot find any other way to fix this. On a result array of 50 results I get sometimes 8, sometimes 20-ish or 30-ish items, depending on the loop design.

This is on Win7 with PHP 7.1.7 (cli) (built: Jul  6 2017 17:04:27) ( ZTS MSVC14 (Visual C++ 2015) x86 ). If you have any good idea, I'll gladly take it.